Open Judge System/Web/OJS.Web/Areas/Administration/Views/Problems/_SubmissionsGrid.cshtml

```diff
@@ -17,8 +18,8 @@
         columns.Bound(model => model.Points);
         columns.Bound(model => model.CreatedOn).Hidden();
         columns.Bound(model => model.ModifiedOn).Hidden();
-        columns.Template(@<text></text>).ClientTemplate(string.Format("<a href='{0}/#= Id #' class='k-button k-button-icontext'>{1}</a>", Url.Action("Update", ControllerName), Resource.Edit));
-        columns.Template(@<text></text>).ClientTemplate(string.Format("<a href='{0}/#= Id #' class='k-button k-button-icontext'>{1}</a>", Url.Action("Delete", ControllerName), Resource.Delete));    
+        columns.Template(@<text></text>).ClientTemplate(string.Format("<a href='{0}/#= Id #' class='k-button k-button-icontext'>{1}</a>", Url.Action("Update", ControllerName), AdministrationGeneralResource.Change));
```

Comment:
The reason is that the specific resources ("Update" and "Delete) can safely apply to the more general Administration resources rather than the concrete Problems resource.